packages/admin-web-angular/src/app/@shared/warehouse/forms/warehouse-manage-tabs/warehouse-manage-tabs.component.ts

```diff
@@ -85,11 +85,7 @@ export class WarehouseManageTabsComponent {
 	}
 
 	get validForm() {
-		return (
-			this.form.valid &&
-			this.contactInfoForm.validForm &&
-			this.paymentsSettingsForm.isPaymentValid
-		);
+		return this.form.valid;
```

Comment:
@YavorGrancharov I don't understand why we removed the other 2 forms checks on saving?

Reply:
For the this.contactInfoForm.validForm validation we already do this validation inside this.form.valid from the Input fields between rows 71 and 85 on  warehouse-manage-tabs.component.ts and it works just fine as shown on the video. For the this.paymentsSettingsForm.isPaymentValid when present on get validForm() no matter how it is used with && or || makes SAVE btn not changing state even if some field on some other section was changed, which forces user/Admin to make changes on payment options or to disable them. My approach was to leave only payment html field validations for ivalid input data without detecting them on SAVE btn but if it is not Ok I can try another approach where both states are detecting any form valid or invalid changes!?